### DISCUSSION TITLE
¿Implementar AZURE?

### DISCUSSION THREAD

**Discussion by Xavi-JC** [Launch-X-Latam/MisionFrontEnd#29] [Ideas]
Se que comander djio que iba a hacer una encuesta para elegir Realk o Viu (Sorry por la terminología, me las estoy aprendiendo) pero y AZURE se queda por fuera o es fijo que se va a ver

**Comment by JosafatJimenezB** [comment]
Para la terminología es REACT y VUE ambos son frameworks con JavaScript los cuales te ayudara a crear paginas y apps web de manera mas fácil y rápida sin repetir tanto código; Azure te va a ayudar a poner tus practicas en linea (deploy) para que cualquiera las pueda ver, esto esta en la ultima semana de la parte de front

**Comment by Emilio17MC** [comment]
React y VUE son librerías para desarrollo Frontend. El despliegue será en Azure, es decir, podemos subir nuestro app Frontend desarrollada en React o VUE a Azure para publicarla y que sea accesible.

**Comment by Allegseu95** [comment]
@Xavi-JC me parece que esta discusión deberia ir en la categoria de Q&A, y con respecto a tu pregunta y con palabras un poco menos técnicas, REACT y VUE nos serviran para hacer la parte visual de nuestras apps y AZURE es donde estará alojado tu app para que sea visible a todo mundo mediante la web.

**Comment by OATS74** [comment]
Disculpa me podrían informar por donde se están viendo la sección o la clase. Ya que por Microsoft team  no veo movimiento no gente

**Reply by Allegseu95** [reply]
Toda la interacción será por GitHub, todo el material se encuentra en este mismo repositorio, en la sección principal hay un README.md https://github.com/Launch-X-Latam/MisionFrontEnd donde detalla todo el contenido de la misión y sus respectivas indicaciones.

Microsoft Teams serán solamente para las explicaciones en vivo (o así lo mencionaron)
